src/components/permissions/form/Rule.vue

```diff
@@ -28,7 +28,7 @@
     <b-col
       cols="12"
       lg="6"
-      class="mb-lg-4 text-nowrap text-truncate"
+      class="mb-lg-4 text-wrap text-truncate"
```

Comment:
Remove text truncate if wrap is used. They are mutually exclusive. 
Similar to you cant have an enabled or disabled button at the same time